frontend/src/components/WorkProcessSection/index.jsx

```diff
@@ -14,43 +14,42 @@ const WorkProcess = () => {
     const { title, circleItem } = workProcessData;
 
     return (
-      <section
-        style={{ borderTopLeftRadius: '200px', marginTop: '5.625rem' }}
-        className="border-4 border-c1000 pt-24 pb-32 bg-c400  mt-20"
-      >
+      <section className="border-4 border-c1000 lg:pt-24 pt-16 pb-32 bg-c400 lg:rounded-tl-md lg:mt-24 mx-auto">
         <Heading
           as="h1"
           fontSize={HEADING_OPTIONS.FONT_SIZES.MEDIUM}
           color={HEADING_OPTIONS.COLOR.PRIMARY}
-          className="text-center mb-16"
+          className="text-center lg:mb-16 mb-16"
         >
           {title}
         </Heading>
         <div className="container relative">
-          <div className=" grid grid-cols-4 items-center justify-center">
+          <div className=" grid  lg:grid-cols-4 sm:grid-cols-1 items-center justify-center ">
             {circleItem.map(item => {
               const { icon, title, id } = item;
               return (
-                <div className="relative z-50 mx-auto" key={id}>
-                  <div className="mx-auto circle__container flex justify-center items-center h-40 w-40 rounded-full parent-circle bg-white hover:bg-c100">
-                    <div className="p-8 border border-dashed rounded-full border-c100 child-circle">
-                      <img src={icon} alt="" />
+                <div className="">
+                  <div className="relative z-50 mx-auto lg:mb-0 mb-12" key={id}>
+                    <div className="mx-auto circle__container flex justify-center items-center lg:h-40 lg:w-40 h-70 w-70 rounded-full parent-circle bg-white hover:bg-c100">
```

Comment:
if you look here you will see that you made the width and the height at the mobile .. more than at the large screens.
please, consider this and don't change any width or height for the circle.